### ISSUE TITLE
rethink organization of teacher resources section now that we are compiling workbook pages for lessons

### ISSUE THREAD

**Issue by flannery-denny** [bootstrapworld/curriculum#1574]
- [x] #1758
- [ ] #1757
- [x] #1725
- [x] #1759
- [x] once everything is building correctly, add some css rules to fix spacing
Currently we are listing pages individually for each lesson on the resources page...

<img width="895" alt="Screen Shot 2023-10-11 at 12 34 56 PM" src="https://github.com/bootstrapworld/curriculum/assets/66691020/21e8dc91-508d-4841-b140-6cf23d73c770">

**Comment by retabak** [comment]
I suppose the organizations of the solutions should mirror the way that we're organizing things on the lesson table? At the moment, I can't think of any reasons to leave it this way? Curious what your thoughts are @flannery-denny , and happy to discuss in real time!

**Comment by flannery-denny** [comment]
@retabak while I'm excited to get this fixed, pushing the conversation off until we've met our jan 1 workbook deadline.

**Comment by flannery-denny** [comment]
@retabak want to make a time for the two of us to meet about this next Tuesday or Wednesday and see if we can put a spec together for ES?

**Comment by retabak** [comment]
ESSENTIALS
- Remove "Blank Image Contracts Page"
- "Contracts" - include both searchable page and PDF - change description
- descriptions in Essentials section are too wordy

STARTER FILES
- Can this link be autogenerated? How would they be associated with the lessons?


![image](https://github.com/bootstrapworld/curriculum/assets/87144723/38445b46-7e1a-4da3-8741-13df9c444317)

![image](https://github.com/bootstrapworld/curriculum/assets/87144723/639bf592-ac48-436e-bfe4-0545d8947d77)

**Comment by flannery-denny** [comment]
@retabak I've removed duplicate starter files from resources sections for `algebra-pyret`, `algebra-wescheme` and `data-science` in this commit: a9ea06dd00240cc91a88729cd7ca4457355f36b3

Wasn't sure what to do with data-science-codap since it doesn't have starter files.

Also, we didn't talk about exercises and solutions. I suggest we put a down arrow in front of each lesson title so that teachers can navigate to a lesson before seeing pages listed. Thoughts?

**Comment by schanzer** [comment]
@flannery-denny and @retabak it looks like everything except for #1757 is finished, and that issue isn't really about the organization of the Teacher Resource page. OK to close?

**Comment by flannery-denny** [comment]
@schanzer that’s fineOn Mar 25, 2024, at 8:07 PM, Emmanuel Schanzer ***@***.***> wrote:﻿
@flannery-denny and @retabak it looks like everything except for #1757 is finished, and that issue isn't really about the organization of the Teacher Resource page. OK to close?
